### PR TITLE
mdlint fixes, add github link to credits, moved multilanguage support and pdf

### PR DESCRIPTION
**Pull request by remkohdev**
* travis.yml requires spaces instead of tabs
* add github profile link to credits for steve martinelli

### CODE REVIEW

**Comment by remkohdev** [comment]
Sorry it got a bit messy, because I added multi-languages support, but first, the rendering didn't come out nice and second the travis script doesnt seem to allow multi-language sub-directories, so I restored the single language and moved the languages/japanese to the root.

File: workshop/README.md [hunk 1]
```diff
@@ -32,31 +18,30 @@ The `Introduction` page has the following sections:
...
-
-## Translations
-
-* [Japanese](README-ja.md) - Thanks to [Toshio Yamashita](https://github.com/developerWorksJP)
+- [John Zaccone](https://github.com/jzaccone)
+- [Jason Kennedy](https://github.com/jkomg)
+- [Steve Martinelli](https://github.com/stevemar)
+- [Remko de Knikker](https://github.com/remkohdev)
 
+en
```
Comment (stevemar) [inline workshop/README.md]: not sure what this is doing here?

**Comment by jzaccone** [comment]
THe gitbook doesn't build. Is this intentional?
Error: ENOENT: no such file or directory, scandir '/Users/johnzaccone/remkointro/workshop/en'

**Comment by jzaccone** [comment]
Links to next lab (at bottom of each lab README) in the jp instructions are broken

File: workshop/README.md (hunk 1 above)
Comment (remkohdev) [reply workshop/README.md]: it shouldn't, i used it to differentiate between 2 files when I was trying to debug the multi-language support, but had to remove multi-language support cause the travis setup doesnt support it i think and the gitbook layout was quite confusing and ugly.

**Pull request merged**